play/src/components/gameinprogress/ChooseAnswer.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
```

Comment:
`GameInProgress` now handles `CHOOSE_CORRECT_ANSWER`, `PHASE_1_DISCUSS`, `CHOOSE_TRICKIEST_ANSWER`, `PHASE_2_DISCUSS.`

It works like so: 
`GameInProgress` - container
`ChooseAnswer` - content for `CHOOSE_CORRECT_ANSWER` and `CHOOSE_TRICKIEST_ANSWER`
`DiscussAnswer` - content for `PHASE_1_DISCUSS` and `PHASE_2_DISCUSS`

see routing in `GameInProgress` for more info